src/CUDAdrv.jl

```diff
@@ -3,21 +3,24 @@ __precompile__()
 module CUDAdrv
 
 using Compat
-import Compat.String
+using Compat.String
 
-const ext = joinpath(@__DIR__, "..", "deps", "ext.jl")
-isfile(ext) || error("Unable to load $ext\n\nPlease run Pkg.build(\"CUDAdrv\") and restart Julia.")
-include(ext)
+const ext = joinpath(dirname(@__DIR__), "deps", "ext.jl")
+if isfile(ext)
+    include(ext)
+else
+    error("Unable to load dependency file $ext.\nPlease run Pkg.build(\"CUDAdrv\") and restart Julia.")
+end
 const libcuda = libcuda_path
 
-include("util/logging.jl")
+include(joinpath("util", "logging.jl"))
```

Comment:
I just double checked this. For example 
```julia
module A
include("b/b.jl")
end
````
produces
```julia
julia> include("C:\\Users\\Mus\\Desktop\\testt\\A.jl")
julia> using A
julia> A.foo()
ERROR: MethodError: no method matching foo()
Closest candidates are:
  foo(::Any) at C:\Users\Mus\Desktop\testt\b/b.jl:1
```

vs.

```julia
module A
include(joinpath("b", "b.jl"))
end
````
produces
```julia
julia> include("C:\\Users\\Mus\\Desktop\\testt\\A.jl")
julia> using A
julia> A.foo()
ERROR: MethodError: no method matching foo()
Closest candidates are:
  foo(::Any) at C:\Users\Mus\Desktop\testt\b\b.jl:1
```

I believe there has been some discussion regarding a more sophisticated filepaths abstraction a la https://github.com/JuliaLang/Juleps/pull/26 etc.